NitroxServer/GameLogic/Players/PersistedPlayerData.cs

```diff
@@ -14,30 +15,36 @@ public class PersistedPlayerData
         public string Name { get; set; }
 
         [JsonProperty, ProtoMember(2)]
-        public List<EquippedItemData> EquippedItems { get; set; } = new List<EquippedItemData>();
```

Comment:
The problem is that protobuf will never be upgradeable if you add variables and Json has no problem with it at all iirc. Just merge wise it can be a bit irritating. I wanted the two fields around EquippedItems and Modules because of context.